jekyll/_cci2/configuration-reference.md

```diff
@@ -102,26 +102,28 @@ jobs:
 ```
 
 ### Machine executor
-Configured by `machine` key which takes a map:
+Configured by using the `machine` key, which takes a map:
 
 Key | Required | Type | Description
 ----|-----------|------|------------
 enabled | Y | Boolean | This must be true in order to enable the `machine` executor.
 {: class="table table-striped"}
 
-As a shorthand you can set the `machine` key to `true`.
+<!-- as of mar 10 2017 this next line is reported as not working, so commenting it out -->
```

Comment:
I've just pushed PR to fix this (https://github.com/circleci/build-agent/pull/305) will update doc as soon as it merged